src/ert/_c_wrappers/enkf/enkf_fs_manager.py

```diff
@@ -57,6 +67,24 @@ def current_case(self) -> "EnkfFs":
     def cases(self) -> List[str]:
         return [x.stem for x in Path(self.storage_path).iterdir() if x.is_dir()]
 
+    def _check_version(self) -> None:
+        """
+        Checks if the file system on disk was created with the same fs version
+        currently in use.
+        """
+        version_file = self.storage_path / FS_VERSION_FILE
+        if not version_file.exists():
+            # If the version file does not exist it uses old fs
+            with open(version_file, "w") as f:
+                json.dump({"version": 0}, f)
+        with open(version_file, "r") as f:
+            version_data = json.load(f)
+        if not version_data["version"] == FS_VERSION:
+            raise ValueError(
+                "Trying to load storage created by fs-version: "
```

Comment:
Write file system version instead of fs-version since users might not be familiar with the `fs` abbreviation?